client/public/index.html

```diff
@@ -58,7 +58,7 @@
       property="og:description"
       content="The MDN Web Docs site provides information about Open Web technologies including HTML, CSS, and APIs for both Web sites and progressive web apps."
     />
-    <meta property="og:image" content="%PUBLIC_URL%/mdn-social-share.png" />
+    <meta property="og:image" content="%BASE_URL%/mdn-social-share.png" />
```

Comment:
Using a new `%BASE_URL%` here because fixing `%PUBLIC_URL%` is tricky and dangerous.